### PR TITLE
Add new voter-reg types to ContentfulEntryLoader

### PR DESCRIPTION
**Pull request by aaronschachter**
### What's this PR do?

This pull request deprecates the hardcoded OVRD Alpha page component, by adding support to display a `VoterRegistrationDriveAction` and/or `VoterRegistrationReferralsBlock` entry from a page's `blocks` multi-value reference field.

I've manually updated the dev and production OVRD campaign action `Page` entry `block` fields with expected `VoterRegistrationDriveAction` and `VoterRegistrationReferralsBlock` entries, as well as the `contentBlock` entries used within the `AlphaPage` component:

Screen Shot 2020-06-01 at 12 01 32 PM

I know this is a big one, but the majority of these changes are updating the GraphQL schema for Cypress, and splitting the existing tests for  `alpha-voter-registration-drive` test into separate tests for each block.

Because it's a biggie, holding off on documentation updates to have less lines to review. Summary of the feature changes made:

* Edits `Page` content type's `blocks` field to allow `VoterRegistrationDriveAction` and `VoterRegistrationReferralsBlock` entries -- I've made this configuration change manually in Contentful

* Modifies `ContentfulEntryLoader`, `ContentfulEntry` components to return expected React components for `voterRegistrationDriveAction` and `voterReferstrationReferralsBlock` entries

* Removes hardcoded AlphaPage component and check for the OVRD campaign path, displaying page `blocks` field content instead

### How should this be reviewed?

Beyond 👀 the code, I've the review app into Aurora -- feel free to login and verify that https://dosomething-phoenix-de-pr-2186.herokuapp.com/us/campaigns/online-registration-drive/action looks as expected.

### Any background context you want to provide?

Once this is merged, I think it'd make sense to get rid of the `Beta` dir within `pages/VoterRegistrationDrivePage` and rename the `BetaPage` component as just `VoterRegistrationDrivePage` (since we no longer have an `Alpha/AlphaPage`)

### Relevant tickets

References [Pivotal #173019820](https://www.pivotaltracker.com/story/show/173019820), [173019860](https://www.pivotaltracker.com/story/show/173019860)

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [x] Added appropriate feature/unit tests.

### CODE REVIEW

File: contentful/content-types/page.js [hunk 1]
```diff
@@ -265,10 +267,13 @@ module.exports = function(migration) {
   });
 
   page.changeFieldControl('metadata', 'builtin', 'entryLinkEditor', {});
-  page.changeFieldControl('authors', 'builtin', 'entryLinksEditor', {});
+
+  page.changeFieldControl('authors', 'builtin', 'entryLinksEditor', {
+    bulkEditing: false,
+  });
```
Comment (katiecrane) [inline contentful/content-types/page.js:273]: this change seems unrelated to the work here, was this intentional?

File: contentful/content-types/page.js [hunk 2]
```diff
@@ -265,10 +267,13 @@ module.exports = function(migration) {
...
 
   page.changeFieldControl('coverImage', 'builtin', 'assetLinkEditor', {
-    helpText: 'The cover image will display on the page before the content',
+    helpText: 'The cover image will display on the page before the content.',
```
Comment (katiecrane) [inline contentful/content-types/page.js:276]: same here, this is just from pulling this from contentful I'm guessing?

File: cypress/integration/voter-registration-drive-action.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      method: 'POST',
+      url: '/api/v2/links',
+      status: 503,
+      response: {},
+    });
+  });
+
+  it('Renders sum of quantity if approved posts query returns results', () => {
+    const user = userFactory();
+    const actionId = faker.random.number();
```
Comment (katiecrane) [inline cypress/integration/voter-registration-drive-action.js:46]: I don't think `actionId` is used here

File: cypress/integration/voter-registration-drive-action.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    });
+
+    cy.authVisitBlockPermalink(user, blockId);
+
+    cy.get('[data-test=total-accepted-quantity-value]').contains('60');
+  });
+
+  it('Renders 0 if approved posts query does not return any results', () => {
+    const user = userFactory();
+    const actionId = faker.random.number();
```
Comment (katiecrane) [inline cypress/integration/voter-registration-drive-action.js:59]: This one appears unused too

File: contentful/content-types/page.js (hunk 1 above)
Comment (aaronschachter) [reply contentful/content-types/page.js:273]: This is a result of running the migration from the Contentful field type on our `master` space. this could have been a change someone made in the admin UI that never got copied into code (the only reason we really need to copy into code is to try to have code match the UI for sake of code reviews)

File: contentful/content-types/page.js (hunk 2 above)
Comment (aaronschachter) [reply contentful/content-types/page.js:276]: Same as https://github.com/DoSomething/phoenix-next/pull/2186#discussion_r433535735 -- this change is per exporting the production type definition per [step 3 of our workflow](https://github.com/DoSomething/phoenix-next/blob/master/docs/development/contentful/workflow.md#3-export-the-changes-as-migrations).

File: cypress/integration/voter-registration-drive-action.js (hunk 3 above)
Comment (aaronschachter) [reply cypress/integration/voter-registration-drive-action.js:46]: Great catch! Removed this and the other one, thanks.

File: cypress/integration/voter-registration-referrals-block.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    cy.get('[data-test=referral-list-item-empty]').should('have.length', 3);
+    cy.get('[data-test=referral-list-item-completed]').should('have.length', 0);
+    cy.get('[data-test=additional-referrals-count]').should('have.length', 0);
+  });
+
+  it('Displays 2 completed icons if user has 2 referrals', () => {
+    const user = userFactory();
+
+    cy.mockGraphqlOp('VoterRegistrationReferrals', {
+      posts: [fakePost('Jesus Q.'), fakePost('Walter S.')],
```
Comment (katiecrane) [inline cypress/integration/voter-registration-referrals-block.js:51]: What about Knox Harrington, the video artist??

**Comment by aaronschachter** [comment]
Thanks for reviewing @katiecrane ! I've verified expected action page on QA: https://qa.dosomething.org/us/campaigns/online-registration-drive/action so this should be in good shape to deploy tomorrow AM, @mendelB 

(Had to change the QA Contentful entry's scholarship end date to avoid the closed campaign view)
